src/Materials/Node/nodeMaterial.ts

```diff
@@ -1659,6 +1659,10 @@ export class NodeMaterial extends PushMaterial {
                 }
             }
 
+            if (merge && this.editorData && this.editorData.locations) {
+                locations.concat(this.editorData.locations);
+            }
+

```

Comment:
I feel this was missing from the previous PR so I added it: is that ok?

Reply:
yep